src/content/docs/reference/utils/Voltage Controller/voltage-controller.md

```diff
@@ -0,0 +1,67 @@
+---

```

Comment:
Rename this file to index.md (update all links to it as well)
This way the route for it is Voltage-Controller not voltage-controller/voltage-controller
Also remove the space in folder name

Reply:
done